src/org/python/core/PyObject.java

```diff
@@ -221,22 +221,44 @@ void proxyInit() {
      * <code>PyObject</code> is to override the standard Java <code>toString</code> method.
      **/
     /*
+     * This was as exposed __str__ method in the original @ExposedMethod annotation
```

Comment:
I would not record the history here. Comments should relate to the current state (which since it is "intutive" could be concise).